src/agents/model_settings.py

```diff
@@ -107,6 +108,10 @@ class ModelSettings:
     """Additional output data to include in the model response.
     [include parameter](https://platform.openai.com/docs/api-reference/responses/create#responses-create-include)"""
 
+    top_logprobs: int | None = None
```

Comment:
this would be used in not only responses, but LiteLLM and Chat Completions, so can you update the following files as well?
- src/agents/models/openai_chatcompletions.py
- src/agents/extensions/models/litellm_model.py